src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1047,18 +1029,15 @@ k
      * @param cc
      */
     canvas_draw_all_aircraft(cc) {
-        // FIXME: How is this different from at line 793?
-        cc.fillStyle = COLOR.LIGHT_SILVER;
-        cc.strokeStyle = COLOR.LIGHT_SILVER;
-        cc.lineWidth = 2;
-
-        // console.time('canvas_draw_all_aircraft')
         for (let i = 0; i < prop.aircraft.list.length; i++) {
+            // color of position dot
+            cc.fillStyle = this.theme.RADAR_TARGET.RADAR_TARGET;
+            cc.lineWidth = 2;
```

Comment:
actually this is reference to an old stroke of the target which was removed, so the linewidth setting can also be removed.

done